drivers/ksz8851snl/Makefile

```diff
@@ -0,0 +1,3 @@
+MODULE = ksz8851snl
```

Comment:
you could drop this line, a siple `include $(RIOTBASE)/Makefile.base` is sufficient for this file, as the make system fill deduct the module name from the path automatically